app/models/user.rb

```diff
@@ -65,7 +65,7 @@ def avatar_url(variant = nil)
     end
   end
 
-  def self.default_avatar_url
+  def default_avatar_url
```

Comment:
This is meant to be a class method, but maybe we could move it to Users Helper instead.

Reply:
The change we want is to turn line 64 to

```ruby
User.default_avatar_url
```

instead of `self.default_avatar_url`

Reply:
I turned line 64 into ```User.default_avatar_url``` and returned line 68 to the original state. 

I did not move the ```default_avatar_url``` method into Users Helper.